ijson/backends/yajl2_c.py

```diff
@@ -0,0 +1,26 @@
+#
+# Contributed by Rodrigo Tobar <rtobar@icrar.org>
+#
+# ICRAR - International Centre for Radio Astronomy Research
+# (c) UWA - The University of Western Australia, 2016
+# Copyright by UWA (in the framework of the ICRAR)
+# All rights reserved
+#
+'''
+Wrapper for _yajl2 C extension module
+'''
+import decimal
+
+from ijson import common
+from ._yajl2 import basic_parse as _basic_parse  # @UnresolvedImport
+from ._yajl2 import parse as _parse  # @UnresolvedImport
+from ._yajl2 import items as _items  # @UnresolvedImport
```

Comment:
Naming nit: I'd prefer `_yajl2` renamed to `pyyajl2` and importing the module itself instead of the individual objects, and refer to them as `pyyajl2.basic_parse`, `pyyajl2.parse` and `pyyajl2.items` from code. This would also remove the need to start their names with `_`.